lib/block.rb

```diff
@@ -0,0 +1,28 @@
+require 'time'
+require 'date'
+require_relative 'reservation'
+
+
+class Block < Reservation
+  attr_reader :block_id, :rooms, :start_date, :end_date
+  COST = 100
+
+  def initialize(block_id, rooms, start_date, end_date)
+    super(start_date, end_date)
+    @block_id = block_id
+    @rooms = rooms
+  end
+
+  def overlap?(start_date, end_date)
+    super(start_date, end_date)
```

Comment:
For this and the method below you can leave them off completely since they are using the `super` method implementation completely